android/PhysicalWeb/app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +33,19 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <activity
+            android:name=".BroadcastActivity"
+            android:label="@string/app_name"
+            android:launchMode="singleTask"
+            android:theme="@android:style/Theme.NoDisplay"
+            android:excludeFromRecents="true"
+            android:taskAffinity="app.broadcast">
```

Comment:
With launchMode="singleTask" there's no reason to set taskAffinity (the activity will always be created in its own task)